src/edu/csus/ecs/pc2/ui/TestResultsPane.java

```diff
@@ -1086,6 +1145,22 @@ public void actionPerformed(ActionEvent e) {
         }
         return selectAllButton;
     }
+
+    private JButton getExecuteAllButton() {
+
+        if (executeAllButton == null) {
+            executeAllButton = new JButton("Execute All");
+        }
+        return executeAllButton;
+    }
+
+    public void addActionListenerToExecuteAll(java.awt.event.ActionListener e) {
+        getExecuteAllButton().addActionListener(e);
+    }
+
+    public void setExecuteAllButton(boolean b) {
```

Comment:
As noted elsewhere, I would recommend changing the name of this method to `setExecuteAllButtonEnabled()`, (or perhaps `setExecuteAllButtonEnabledState()` or `enableExecuteAllButton()`) for clarity.